### PR TITLE
Minor bugs fixed

### PR DESCRIPTION
**Pull request by igorsegallafa**
- Fixed some warnings caused on Sample2D and changed the code to pass structs by reference, instead of by value (when possible)
- Fixed a bug where CharacterMap::Load is always returning false
- Fixed a bug on AnimatedSprite2D animation (alpha animation was not working properly)

### CODE REVIEW

File: Source/Urho3D/Urho2D/AnimatedSprite2D.cpp [hunk 1]
```diff
@@ -496,7 +496,11 @@ void AnimatedSprite2D::UpdateSourceBatchesSpriter()
         vertex2.uv_ = textureRect.max_;
         vertex3.uv_ = Vector2(textureRect.max_.x_, textureRect.min_.y_);
 
-        vertex0.color_ = vertex1.color_ = vertex2.color_ = vertex3.color_ = color;
+		Color finalColor;
+		finalColor.FromUInt( color );
+		finalColor.a_ = info.alpha_;
```
Comment (weitjong) [inline Source/Urho3D/Urho2D/AnimatedSprite2D.cpp]: Wrong spacing convention: wrong indent and extra space around method parameter.

File: Source/Samples/Utilities2D/Sample2D.h [hunk 2]
```diff
@@ -61,17 +61,17 @@ class Sample2D : public Object
     ~Sample2D() override = default;
 
     /// Generate physics collision shapes from the tmx file's objects located in tileMapLayer.
-    void CreateCollisionShapesFromTMXObjects(Node* tileMapNode, TileMapLayer2D* tileMapLayer, TileMapInfo2D info);
+    void CreateCollisionShapesFromTMXObjects(Node* tileMapNode, TileMapLayer2D* tileMapLayer,const TileMapInfo2D& info);
```
Comment (eugeneko) [inline Source/Samples/Utilities2D/Sample2D.h]: Space, please

Comment (igorsegallafa) [reply Source/Samples/Utilities2D/Sample2D.h]: Fixed.

**Comment by igorsegallafa** [comment]
Fixed on latest commit.

**Comment by weitjong** [comment]
Thanks!

**Pull request merged**